src/js/themes/hpe.js

```diff
@@ -1448,33 +1490,37 @@ export const hpe = deepFreeze({
   },
   text: {
     xsmall: {
+      // weight needs to be modified at the size level
       size: '14px',
-      height: '20px',
-      maxWidth: '326px',
+      height: '18px',
+      maxWidth: '379px',
```

Comment:
Why are we boosting maxWidth to align with the "small" case? I was expecting the maxWidth to be generally proportional to the font size.

Reply:
That's a copy-paste error on my side. I will fix.

With the shift to thinner font weights, I think we should re-evaluate the maxWidths throughout to ensure they fit to the readability line length recommendations. Item in the [todo list](https://github.com/grommet/hpe-design-system/issues/3020#issue-1448456130).